datasets.py

```diff
@@ -158,3 +159,43 @@ def get_cast(clean=False):
     cast_df.to_csv("./clean_datasets/clean_cast.csv", index=False)
     
     return cast_df
+
+def get_actors():
+    try:
```

Comment:
Can you follow the same convention used for the other datasets have a clean flag in the function that declares whether or not to reclean the dataset

Reply:
Got it, will change.